### PR TITLE
Blog post for fitting the correlation function

### PR DESCRIPTION
**Pull request by ChayaSt**
@jchodera , can you take a pass through this?

Thanks!

### CODE REVIEW

File: _posts/2016-08-15-fitting-the-correlation-function.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+author: chayast
```
Comment (jchodera) [inline _posts/2016-08-15-fitting-the-correlation-function.md:3]: You'll need to add yourself [here](https://github.com/ChayaSt/klogw/blob/gh-pages/_data/authors.yml) too!

File: _posts/2016-08-15-fitting-the-correlation-function.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+date: 2016-08-15
+title: Fitting the Correlation Function
+published: true
+---
+
+Single molecule fluorescent experiments provide a mean to study subtle, time-dependent movement, such as structural
```
Comment (jchodera) [inline _posts/2016-08-15-fitting-the-correlation-function.md]: Single molecule _fluorescence_ experiments provide a _means_ to study

File: _posts/2016-08-15-fitting-the-correlation-function.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+rearrangements, which cannot easily be studied in bulk. Such experiments result in many trajectories of the observed
+intensity over time from which useful information needs to be extracted. The trajectories can be used to calculate the
+correlation function that describes the correlation between points as a function of lag time. The correlation function
+contains all of the essential information of the trajectories in compact form and can be used to extract relaxation
+timescales in a model free way. Since the timescales for slow processes are invariant to the system, these timescales
+can be compared to timescales extracted through other means as a way to invalidate proposed kinetic models.
+
+The theoretical correlation function can be expressed as a sum of exponential $$ \mathbf C(t) = \sum_{i=1}^N A_i e^{-t/\tau_i } $$. The
```
Comment (jchodera) [inline _posts/2016-08-15-fitting-the-correlation-function.md]: Maybe cite the "dynamical fingerprints" paper here too, or some of the Markov state model reviews which show how correlation functions can be computed from MSMs (I think Bettina Keller or Nuria Plattner and Frank Noé wrote one)?

**Comment by jchodera** [comment]
I made some comments, but this looks good otherwise!  Thanks!

**Comment by jchodera** [comment]
Is this ready to go, @ChayaSt ?

**Comment by ChayaSt** [comment]
Yes.

**Comment by jchodera** [comment]
It's up!

http://klogw.org/

Maybe we can add a graph too?

**Pull request merged**